libs/core/src/storage/noopstorage/noopstorage.service.ts

```diff
@@ -0,0 +1,8 @@
+import { DaffPersistenceService } from "../persistence.interface";
+
+export class DaffNoopStorageService implements DaffPersistenceService {
```

Comment:
This should be decorated with `@Injectable`.

Reply:
done